src/Kunstmaan/AdminBundle/Resources/config/routing_fos.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+    <route id="fos_user_profile_show" path="/" methods="GET">
+        <default key="_controller">fos_user.profile.controller:showAction</default>
```

Comment:
FosUserBundle has a ProfileController. I miss this now.

Reply:
Yes we will remove this route/action as we neve supported it in the CMS UI anyway.

Reply:
And the cms already had its own edit action seperate from the FOS one.

Reply:
This should be mentioned in the upgrade then, because we overwrite the ProfileController from FosUserBundle to add some extra functionalities.

Reply:
We only lose the 'showaction' which was never used or officially supported by our UI so I don't see why it is so important to mention this? The edit action has indeed been overwritten by our own edit action in a different controller and has been the same for years.

Reply:
We use that showAction on the frontend. But if there is a mention that fosuserbundle is removed (or something different) then that is enough.

Reply:
ok 👍

Reply:
The idea is that from 5.8 our code and the fos bundle will run next to each other and you can choose between them. so the dependency isn't fully removed yet. that will happen in 6.0